templates/test.yaml

```diff
@@ -26,5 +26,7 @@ spec:
         {{include "rhtap.namespace.test" . | indent 8}}
         {{include "rhtap.openshift-gitops.test" . | indent 8}}
         {{include "rhtap.openshift-pipelines.test" . | indent 8}}
+        {{include "rhtap.tas.test" . | indent 8}}
```

Comment:
Use the full product name instead of the acronym (cf. templates/configure.yaml).